docker-compose.yml

```diff
@@ -2,6 +2,8 @@ version: '3'
 services:
   server:
     image: app:build
+    build: 
+      context: .
```

Comment:
Just noticed, that this will need to be applied to the other services here, too.

Reply:
This will also need the [target](https://docs.docker.com/compose/compose-file/#target) parameter in the docker-compose.yml to be able to use the right stage.

Reply:
Hey @jezdez,

I can add the target field, but I think I want to hold off on that for now with docker-compose because I will need to up the version to 3.4 and the max supported version for docker-compose with circle CI 2.0 is 3.2.

The reason why I want to hold of is mainly because migrating to circle CI 2.1 adds a few new things that I think could cause some breaking changes and we might want to separate that work into a separate small pr.